modules/kubernetes/pages/service-certs.adoc

```diff
@@ -0,0 +1,164 @@
+// Type: assembly
+[id="encrypting-traffic-pod-router"] 
+= Encrypting traffic from a pod to the Skupper router
+
+This section describes how to apply certificates to encrypt the traffic within a cluster.
+
+By default:
+
+* {Skupper-name} creates certificates to establish links between sites using mutual TLS.
+This ensures that traffic between sites is encrypted. To use custom certificates for traffic between sites, see xref:index.adoc[].
+
+* Network traffic between pods and the Skupper router is not encrypted. 
+
+You can use certificates to encrypt traffic between pods and the Skupper router as follows:
+
+* xref:skupper-generated-certs[] - does not require user to provide certificates.
+* xref:user-provided-certs[] - requires that the certificate authority, and the signed certificates, are distributed across all sites.
+
+NOTE: With both procedures, you reference the certificates when you expose the service on the {service-network}.
+If you do not reference certificates, the traffic between pods and the Skupper router is unencrypted.
+
+// Type: procedure
+[id="skupper-generated-certs"]
+== Exposing services on the {service-network} with Skupper-generated certificates
+
+A {skupper-name} installation includes a certificate authority which can generate certificates that can be used to encrypt traffic from the pod to the Skupper router.
+This procedure describes how to use those certificates in your {service-network}.
+
+.Prerequisites
+
+* Access to Kubernetes site where the service is exposed
```

Comment:
```suggestion
* Access to the Kubernetes site exposing the service
```
Removed PV.

Reply:
added 'the', but we don't know who exposed the service, so need to leave PV